modules/storage.js

```diff
@@ -0,0 +1,33 @@
+export default class Storage {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  // Event: Adds Books
+
+  static addBook(book) {
+    const books = Storage.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+    window.location.reload();
+  }
+
+  // Event remove a Book
+  static removeBook(title) {
+    const books = Storage.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
Please consider reviewing your remove books code as they are not updating the local storage and due to that reason it deleted the book when we click the remove button but when we load the page it still persists there. Consider reviewing that.

Reply:
> Please consider reviewing your remove books code as they are not updating the local storage and due to that reason it deleted the book when we click the remove button but when we load the page it still persists there. Consider reviewing that.

Hi @ABDULALI3468 , a big thank you for your feedback. However I have fixed the issue rather in [ui.js](https://github.com/ernestmusong/Awesome-books-ES6/blob/Add-ES6-modules/modules/ui.js#:~:text=%7D-,static%20deleteBook(elem%2C%20id)%20%7B,%7D,-static%20clearFields) and in [index.js](https://github.com/ernestmusong/Awesome-books-ES6/blob/Add-ES6-modules/index.js#:~:text=document.querySelector(%27%23container,%7D)%3B). It works now.